tests/x11/network/yast2_network_use_nm.pm

```diff
@@ -14,7 +14,7 @@
 
 use base 'x11test';
 use y2_module_guitest 'launch_yast2_module_x11';
-use y2_installbase;
+use y2_base;
```

Comment:
This is very strange, what we do here, `y2_module_guitest` should include post fail hook, as for ncurses we have `y2_module_consoletest`. As it's out of scope here, could you please file a ticket to make sure we have correct structure being used?

Reply:
https://progress.opensuse.org/issues/91761